gordon_janitor/main.py

```diff
@@ -177,7 +177,7 @@ def run(config_root):
 
     logging.info(f'Loaded {len(plugin_names)} plugins: {plugin_names}')
     logging.info(f'Starting gordon janitor v{version}...')
-    loop = asyncio.get_event_loop()
+    loop = asyncio.new_event_loop()
```

Comment:
As we discussed, this can lead to funky behavior if we don't use the default loop (and therefore [set it as the loop](https://docs.python.org/3/library/asyncio-eventloops.html#asyncio.AbstractEventLoopPolicy.new_event_loop)). In other places, we use `asyncio.get_event_loop` ([one example](https://github.com/spotify/gordon-gcp/blob/develop/src/gordon_gcp/clients/auth.py#L138-L139)), which returns a different loop than this loop:

```py
>>> loop = asyncio.get_event_loop()
>>> new_loop = asyncio.new_event_loop()
>>> default_loop = asyncio.get_event_loop()  # again to show that new_loop is not default
>>> id(loop)
4527663088
>>> id(new_loop)
4538243840
>>> id(default_loop)
4527663088
>>> asyncio.set_event_loop(new_loop)
>>> another_loop = asyncio.get_event_loop()  # returns the new loop
>>> id(another_loop)
4538243840
```

We'd do this if we explicitly needed more than one loop to switch contexts between. We'd also have to use `set_event_loop`, and since we're only needing one loop, it just puts us back to what `get_event_loop` does for us.

Reply:
Convention is good enough for me. I wasn't aware we were actually expecting the loop to be accessible in Janitor. I'll switch it back.